From 5611801b8bafa310f255016792d02ef6dd26dc33 Mon Sep 17 00:00:00 2001
From: Don Darling <don.osc2@gmail.com>
Date: Mon, 12 Apr 2010 13:29:41 -0500
Subject: [PATCH 05/12] Try progressive mode first for DM6467T.

The v4l2src driver typically tries to initialize a capture device in interlaced
mode first, and if that fails it tried progressive mode.  On DM6467T, only
the component capture is supported and the driver requires that we get this
setting right on the first try -- so change the setting to try progressive
mode first on DM6467T.
---
 sys/v4l2/gstv4l2object.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/sys/v4l2/gstv4l2object.c b/sys/v4l2/gstv4l2object.c
index 48af22c..c9c4fed 100644
--- a/sys/v4l2/gstv4l2object.c
+++ b/sys/v4l2/gstv4l2object.c
@@ -1863,7 +1863,7 @@ gst_v4l2_object_set_format (GstV4l2Object * v4l2object, guint32 pixelformat,
    * correctly the first time we call S_FMT, based on the capture device we are
    * using.  Override settings here where appropriate.
    */
-  #if defined(Platform_dm6467)
+  #if defined(Platform_dm6467) || defined(Platform_dm6467t)
     format.fmt.pix.field = V4L2_FIELD_NONE;
   #endif
 
-- 
1.6.3.3

